composer.json

```diff
@@ -2,7 +2,7 @@
     "name": "jpay/module-payments",
     "description": "Official Jumia Pay Payments Extension for Magento2",
     "type": "magento2-module",
-    "version": "1.0.6",
+    "version": "1.1.0",
```

Comment:
Yes. it´s not exactly related, you are right, but for releasing the plugin, we must increment the plugin version on this file. This could be a separate commit before the release, but since this is a very small change, I decided to put everything in the same PR :)